app/assets/uswds/_uswds-theme.scss

```diff
@@ -1,13 +1,36 @@
-/*
-----------------------------------------
-USWDS with settings overrides
-----------------------------------------
-Uncomment the following lines and add a list of changed settings
-in the form $setting: value,
-----------------------------------------
-*/
```

Comment:
I'd like to keep this comment and maybe remove the "Uncomment the following lines" portion so it just says "Add a list of changed settings in the form $setting: value." for inline documentation